deployments/deployments.json

```diff
@@ -242,8 +242,8 @@
       "sender": "0x95fb5cb304508d74d855514d7bc9bda75c304ce2",
       "core": {
         "address": "0xee4ea8d358473f0fcebf0329feed95d56e8c04d7",
-        "height": 22242649,
-        "commit": "33462cd11c9fc68deab4c1d6db3dada33e1259ef"
+        "height": 22273483,
```

Comment:
heights shouldn't update, we need to fix the script I guess?

Reply:
oof, it's already been updated a couple of time I believe

Reply:
got it, we fetch the initialized height of the impl instead of proxy